src/packages/@ncigdc/modern_components/ProjectCounts/ProjectCountsDataCategory.js

```diff
@@ -132,10 +153,62 @@ export default compose(
           style: { textAlign: 'right' },
         },
         {
+          key: 'case_count_meter',
+          title: (
+            <Link
+              pathname="/repository"
+              query={{
+                filters: makeFilter([
+                  {
+                    field: 'project.project_id',
```

Comment:
this field actually needs to be `cases.project.project_id` since we're making this filter on the case index.